src/com/amazon/ion/benchmark/Main.java

```diff
@@ -44,7 +44,7 @@ public class Main {
 
         + "  ion-java-benchmark run-suite (--test-ion-data <file_path>) (--benchmark-options-combinations <file_path>) <output_file>\n"
 
-        + "  ion-java-benchmark compare (--benchmark-result-previous <file_path>) (--benchmark-result-new <file_path>) <output_file>\n"
+        + "  ion-java-benchmark compare (--benchmark-result-previous <file_path>) (--benchmark-result-new <file_path>)\n"
```

Comment:
Was `<output_file>` not used? Or did you determine that it is not useful, and we should always write to `stdout`?

Reply:
Yes, `<output_file>` is not useful for now. We will write out the regression results.